webapp/queue.yaml

```diff
@@ -0,0 +1,4 @@
+queue:
+- name: results-arrival
+  target: processor
+  rate: 1/m
```

Comment:
Oops, I didn't mean to set the rate. It's the `max_concurrent_requests` that I wanted to change because I use manual scaling for the Flex service.

Removed `rate` and added `max_concurrent_requests` (set to 1 for now).